references/cli.md

```diff
@@ -523,6 +526,9 @@ Options:
   --profile TEXT  Profile name to use.
   --user FILE     User file path to use (default ~/.substra-user).
   --verbose       Enable verbose mode.
+  --yaml          Display output as yaml.
+  --json          Display output as json.
+  --pretty        Pretty print output  [default: True]
```

Comment:
These options are irrelevant for describe.

Reply:
You're right, but I have to remove the output_format option from the `click_global` method to remove it from the `--help`, should we really do that? 🤔

Reply:
We could have 2 decorators, one that includes the output format and one that doesn't.

Reply:
Yes, it could be cleaner this way, I'll work on it!